contracts/MemeFactory.sol

```diff
@@ -286,10 +289,79 @@ abstract contract MemeFactory {
         emit Collected(msg.sender, memeToken, allocation);
     }
 
+    /// @dev Create a new meme token.
```

Comment:
This was already checked when shuffling functions, somehow the reference PR does not point to the correct one.